.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## Description
+
+Thank you for your contribution to the PySyft repository.
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
+List any dependencies that are required for this change.
+
+If your pull request closes a GitHub issue, then set its number below.
+
+Fixes # (issue)
+
+## Type of change
+
+Please mark options that are relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
+Please also add the name of test files added/modified which have the tests for your contribution.
```

Comment:
The list of files changed is probably overkill, because I can see that from the diff. As long as people have added tests for their changes (as listed on the `Checklist` below) that probably covers 95% of what's useful as a PR reviewer. Since we run the repo tests automatically via Github Action, whatever tests get added need to pass in that environment.

I'd probably drop the `How has this been tested` section for now.